examples/viz_recalibrate_readme.py

```diff
@@ -0,0 +1,89 @@
+"""
+Examples of code for recalibration.
```

Comment:
Can we add onto this and say explicitly it produces the example in the README.

Reply:
added!